pkg/config/config.go

```diff
@@ -34,6 +34,7 @@ type Node struct {
 	Hostname                string                        `yaml:"hostname" jsonschema:"required,description=Hostname of the node"`
 	IPAddress               string                        `yaml:"ipAddress,omitempty" jsonschema:"required,example=192.168.200.11,description=IP address where the node can be reached, can also be a comma separated IP addresses"`
 	ControlPlane            bool                          `yaml:"controlPlane" jsonschema:"description=Whether the node is a controlplane"`
+	GrubUseUKICmdline       *bool                         `yaml:"grubUseUKICmdline,omitempty" jsonschema:"description=Whether to use UKI cmdline for grub configuration. If not specified, Talos 1.12+ defaults to true, older versions default to false"`
```

Comment:
This shouldn't be required, it's already solved by https://github.com/budimanjojo/talhelper/commit/11ea025150391413629a3b2d5d09411e7c701798